jf_agent/main.py

```diff
@@ -247,8 +247,8 @@ def obtain_config(args):
     jira_exclude_project_categories = set(jira_config.get('exclude_project_categories', []))
     jira_issue_jql = jira_config.get('issue_jql', '')
 
-    git_config = yaml_config.get('git', yaml_config.get('bitbucket', {}))
```

Comment:
Does this mean that existing config files using the `bitbucket` section will stop working?

Reply:
ah, that's a good point. i should make this backwards compatible then.